File: desktop/src/com/obstacle/avoid/desktop/DesktopLauncher.java
package com.obstacle.avoid.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.obstacle.avoid.ObstacleAvoidGame;
import com.obstacle.avoid.config.GameConfig;

public class DesktopLauncher {
	public static void main (String[] arg) {
		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
		config.width = (int) GameConfig.INSTANCE.getWIDTH();
		config.height = (int) GameConfig.INSTANCE.getHEIGHT();
		new LwjglApplication(new ObstacleAvoidGame(), config);
	}
}

File: desktop/src/com/obstacle/avoid/desktop/DesktopLauncher.kt
package com.obstacle.avoid.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.obstacle.avoid.ObstacleAvoidGame
import com.obstacle.avoid.config.GameConfig

object DesktopLauncher {
    @JvmStatic fun main(arg: Array<String>) {
        val config = LwjglApplicationConfiguration()
        config.width = GameConfig.WIDTH.toInt()
        config.height = GameConfig.HEIGHT.toInt()
        LwjglApplication(ObstacleAvoidGame(), config)
    }
}
